test-clients/src/main/java/com/hedera/services/bdd/suites/contract/ContractUpdateSuite.java

```diff
@@ -50,281 +53,305 @@
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
 
 public class ContractUpdateSuite extends HapiApiSuite {
-	private static final Logger log = LogManager.getLogger(ContractUpdateSuite.class);
+    private static final Logger log = LogManager.getLogger(ContractUpdateSuite.class);
 
-	private static final long defaultMaxLifetime =
-			Long.parseLong(HapiSpecSetup.getDefaultNodeProps().get("entities.maxLifetime"));
-	private static final long ONE_DAY = 60 * 60 * 24;
-	private static final long ONE_MONTH = 30 * ONE_DAY;
+    private static final long defaultMaxLifetime =
+            Long.parseLong(HapiSpecSetup.getDefaultNodeProps().get("entities.maxLifetime"));
+    private static final long ONE_DAY = 60 * 60 * 24;
+    private static final long ONE_MONTH = 30 * ONE_DAY;
 
-	public static void main(String... args) {
-		new ContractUpdateSuite().runSuiteAsync();
-	}
+    public static void main(String... args) {
+        new ContractUpdateSuite().runSuiteAsync();
+    }
 
-	@Override
-	public boolean canRunAsync() {
-		return true;
-	}
+    @Override
+    public boolean canRunAsync() {
+        return true;
+    }
 
-	@Override
-	protected List<HapiApiSpec> getSpecsInSuite() {
-		return List.of(
-				updateWithBothMemoSettersWorks(),
-				updatingExpiryWorks(),
-				rejectsExpiryTooFarInTheFuture(),
-				updateAutoRenewWorks(),
-				updateAdminKeyWorks(),
-				canMakeContractImmutableWithEmptyKeyList(),
-				givenAdminKeyMustBeValid(),
-				fridayThe13thSpec()
-		);
-	}
+    @Override
+    protected List<HapiApiSpec> getSpecsInSuite() {
+        return List.of(
+                updateWithBothMemoSettersWorks(),
+                updatingExpiryWorks(),
+                rejectsExpiryTooFarInTheFuture(),
+                updateAutoRenewWorks(),
+                updateAdminKeyWorks(),
+                canMakeContractImmutableWithEmptyKeyList(),
+                givenAdminKeyMustBeValid(),
+                fridayThe13thSpec(),
+                updateDoesNotChangeBytecode()
+        );
+    }
 
-	private HapiApiSpec updateWithBothMemoSettersWorks() {
-		String firstMemo = "First";
-		String secondMemo = "Second";
-		String thirdMemo = "Third";
-		return defaultHapiSpec("UpdateWithBothMemoSettersWorks")
-				.given(
-						newKeyNamed("adminKey"),
-						contractCreate("contract")
-								.adminKey("adminKey")
-								.entityMemo(firstMemo)
-				).when(
-						contractUpdate("contract")
-								.newMemo(secondMemo),
-						contractUpdate("contract")
-								.newMemo(ZERO_BYTE_MEMO)
-								.hasPrecheck(INVALID_ZERO_BYTE_IN_STRING),
-						getContractInfo("contract").has(contractWith().memo(secondMemo))
-				).then(
-						contractUpdate("contract")
-								.useDeprecatedMemoField()
-								.newMemo(thirdMemo),
-						getContractInfo("contract").has(contractWith().memo(thirdMemo))
-				);
-	}
+    private HapiApiSpec updateDoesNotChangeBytecode() {
```

Comment:
Lets move this test to the bottom, since you changed the formatting and we cannot see clearly what has changed.